core/src/test/java/org/jdbi/v3/core/mapper/RowViewMapperTest.java

```diff
@@ -34,11 +34,11 @@ public class RowViewMapperTest {
 
     @Test
     public void testRowViewMap() {
-        Handle h = h2Extension.getSharedHandle();
         final Map<Integer, String> expected = new HashMap<>();
         expected.put(1, "SFO");
         expected.put(2, "OAK");
         expected.put(3, "YYZ");
+        Handle h = h2Extension.getSharedHandle();
```

Comment:
why did this get moved?

Reply:
`VariableDeclarationUsageDistance` check, same as above